go-server-server/go/default.go

```diff
@@ -1445,6 +1456,9 @@ func ConfigVrfVrfIdRoutesPatch(w http.ResponseWriter, r *http.Request) {
             if r.Profile != "" {
                 route_map["profile"] = r.Profile
             }
+            if r.Persistent == "false" {
```

Comment:
At line 1434, it will continue the operation without setting if the route is identical. So how do you reset this `route_map["refresh"] `attribute when client refreshes the route? Can you please add the test?